tests/test_flo2d.cpp

```diff
@@ -898,6 +899,59 @@ TEST( MeshFlo2dTest, pro_16_02_14 )
   MDAL_CloseMesh( m );
 }
 
+TEST( MeshFlo2dTest, mesh1D )
+{
+  std::string path = test_file( "/flo2d/channel/CADPTS.DAT" );
+  EXPECT_EQ( MDAL_MeshNames( path.c_str() ), "FLO2D:\"" + path + "\":" + "mesh1d" );
+  std::string uri1d = "FLO2D:\"" + path + "\":" + "mesh1d";
+
+  MDAL_MeshH m = MDAL_LoadMesh( uri1d.c_str() );
+  ASSERT_NE( m, nullptr );
+  MDAL_Status s = MDAL_LastStatus();
+  EXPECT_EQ( MDAL_Status::None, s );
+
+  // ///////////
+  // Vertices
+  // ///////////
+  int v_count = MDAL_M_vertexCount( m );
+  EXPECT_EQ( v_count, 66 );
+  EXPECT_TRUE( MDAL::equals( getVertexXCoordinatesAt( m, 10 ), 1361905 ) );
+  EXPECT_TRUE( MDAL::equals( getVertexYCoordinatesAt( m, 10 ), 73194.93 ) );
+
+  // ///////////
+  // Edges
+  // ///////////
+  int e_count = MDAL_M_edgeCount( m );
+  EXPECT_EQ( 65, e_count );
+  std::vector<int> startIndexes;
+  std::vector<int> endIndexes;
+  getEdgeVertexIndices( m, e_count, startIndexes, endIndexes );
+  EXPECT_EQ( startIndexes[10], 10 );
+  EXPECT_EQ( endIndexes[10], 11 );
+
+  // ///////////
+  // Datasets
+  // ///////////
+  ASSERT_EQ( 13, MDAL_M_datasetGroupCount( m ) );
+  MDAL_DatasetGroupH g = MDAL_M_datasetGroup( m, 11 );
+  ASSERT_NE( g, nullptr );
```

Comment:
add test for group name , in case in future something is added, we know which group was tested